### PR TITLE
Display feature id

### PR DESCRIPTION
**Pull request by jingsam**

### CODE REVIEW

File: views/vector.ejs [hunk 1]
```diff
@@ -127,13 +127,15 @@ function renderLayer(layerId) {
   return '<div class="mbview_layer">' + layerId + '</div>';
 }
 
-function renderProperties(feature) {
+function renderProperties(feature) {console.log(feature)
```
Comment (andrewharvey) [inline views/vector.ejs]: is that console.log still needed?

**Comment by andrewharvey** [comment]
Closes #73

**Comment by jingsam** [comment]
What should I do to make this PR to be merged?

**Comment by jingsam** [comment]
Closes #73

**Comment by ericfischer** [comment]
Looks good to me. Thanks!

**Pull request merged**